apps/electrometer/values-tucson-teststand.yaml

```diff
@@ -0,0 +1,21 @@
+namespace: calsys
+image:
+  repository: ts-dockerhub.lsst.org/electrometer
+  tag: c0039
+  pullPolicy: Always
+env:
+  LSST_DDS_PARTITION_PREFIX: tucson
+  LSST_SITE: tucson
+  S3_ENDPOINT_URL: https://s3.tuc.lsst.org
+  RUN_ARG: --simulate
```

Comment:
Remove this line from the config and create a simulation configs for each electrometer with the given index.